config.py

```diff
@@ -53,10 +60,12 @@
         "read_interval": 3,
         "name": "GPS",
     },
-    "RP_CPU_TEMP": {
+    "RP_CPU_TEMPERATURE": {
         "priority": 2,
         "read_interval": 5,
-        "name": "RP_CPU_TEMP",
+        "name": "RP_CPU_TEMPERATURE",
+        "warning_temperature": 80,
+        "alert_temperature": 90,
```

Comment:
warning, alert, max, its a bit too much innit? why not an warning treshold and a max which would be the limit
if its not already what it means ..

Reply:
In this case, there's no max temperature, so I believe you might be confusing things with the second PR.

In any case, it's something I plan to look into as I work on error checking soon. I agree it might be a bit confusing + we might not need all these checks. Also (as you said if I understood correctly) it could be interesting to only store the max temperature, and have a global constant percentage for warning and critical temperatures relative to the max temperature (or battery levels).